Tribler/community/dht/routing.py

```diff
@@ -0,0 +1,192 @@
+import time
+import random
+import binascii
+
+from threading import RLock
+from socket import inet_aton
+
+import datrie
```

Comment:
Did you check whether `datrie` is available in the debian repositories? Also, please update the documentation and add instructions on how to install this new dependency 👍

Reply:
Yup, it's in the repositories, and I just updated the docs.